src/antsibull_build/versions.py

```diff
@@ -333,3 +339,19 @@ def find_latest_compatible(
         )
 
     return reduced_versions
+
+
+def load_all_dependency_files(
+    deps_dir: os.PathLike[str] | str,
+    *,
+    accept_deps_file: t.Callable[[os.PathLike[str] | str, str], bool] | None = None,
+) -> dict[str, DependencyFileData]:
```

Comment:
This function could maybe use a docstring; you have to read almost the entire function to figure out what the `accept_deps_file` key function is actually called with.